constant.go

```diff
@@ -0,0 +1,46 @@
+package backoff
+
+import (
+	"time"
+
+	"github.com/cenkalti/backoff"
+)
+
+func NewConstant(maxWait, maxInterval time.Duration) backoff.BackOff {
```

Comment:
This is taken from `e2e-harness`. Will replace it there with the lab once it lands.